youtube-channels.csv

```diff
@@ -616,3 +616,4 @@ UCdcJ5fWasFdTD0Wml1LKKVw, LA Kistoc, https://www.youtube.com/@lakistoc
 UClD0ybAveASPVSkTqV3x_nA, Ipomea, https://www.youtube.com/@ipomea2662
 UC23oAF7cZWw1pjkwz4f62lw, Housepedia Music, https://www.youtube.com/@HousepediaMusic
 UCoVTIYlXPcm15lq8OeP_W3A, Recordeep, https://www.youtube.com/@recordeep2877
+UCUd_NlUON9ApwV8llL4gHtg, Wandering Endlessly, https://www.youtube.com/@wanderingendlessly
```

Comment:
this is already present on line 373
https://github.com/mirrorfm/data/blob/master/youtube-channels.csv#L373

here's the playlist
https://open.spotify.com/playlist/2DTdmsVcLiAxLicBsurw9o?si=3e680cce55a94439